src/app/model/ChessEngine.ts

```diff
@@ -27,7 +27,12 @@ export class ChessEngine implements ChessModel {
       throw new Error('Piece can not move to given square.');
     }
 
-    const pieceWasMoved: PieceWasMoved = { eventType: 'PieceWasMoved', piece: chosenPiece, from: squareFrom, to: squareTo };
+    const pieceWasMoved: PieceWasMoved = {
+      eventType: 'PieceWasMoved',
+      piece: chosenPiece,
+      from: squareFrom,
+      to: squareTo,
+    };
```

Comment:
Jak to się zadziało? Tu jest tylko format, a przecież prettier i husky powinien do tego nie dopuścić.
Ciekawe

Reply:
hmmm, nie wiem, ale w sumie tak w kolumnie wygląda to dobrze :-) ... może nie tutaj na githubie, ale w kodzie, jest elegancko